common.blocks/page/page.js

```diff
@@ -33,6 +33,17 @@ provide(BEMDOM.decl(this.name, {
                     this.unbindFromWin('scroll', this._disableScroll);
                     this.bindToWin('scroll', this._onScroll);
                 }, this);
+
+                //block scrolling while popup open
```

Comment:
if the only purpose of this piece of code is to block scrolling, it should be consistent with the way it's done above for `video`. if your solution solves it better, please update code above, otherwise use old code for `modal`